internet.go

```diff
@@ -243,9 +243,10 @@ func (i Internet) HTTPMethod() string {
 // Query returns a fake query for Internet
 func (i Internet) Query() string {
 	lorem := i.Faker.Lorem()
+	boolean := i.Faker.Boolean().Bool()
 	query := "?" + lorem.Word() + "=" + lorem.Word()
 	for j := 0; j < i.Faker.IntBetween(1, 3); j++ {
-		if i.Faker.Bool() {
+		if boolean {
```

Comment:
considering that this variable is just used here can you replace the `if boolean {` with `if i.Faker.Boolean().Bool() {`